code/network/multi.h

```diff
@@ -366,6 +366,16 @@ class player;
 
 #define MAX_SHIPS_PER_PACKET		64			// Number of ships in a STATS_MISSION_KILLS or STATS_ALLTIME_KILLS packet
 
+// campaign and mission description packet post-header type info
+#define DESCRIPT_MESSAGE_REQUEST	(1 << 0)
+#define DESCRIPT_PACKET_MESSAGE_1	(1 << 1)
+
+#define MULTI_MAX_DESC_LEN			2*(MAX_PACKET_SIZE - 10)
```

Comment:
Okay, I see the difference now, but it's just confusing. `MULTI_MAX_DESC_LEN` vs `MULTI_PACKET_DESC_LEN`, plus more magic numbers. There should be a less confusing way to handle that.

Reply:
I hope you meant a better name would be sufficient.

The one number I could not figure out from the original code was that they cut off 10 bytes for the packet.